pandas/tests/indexes/test_base.py

```diff
@@ -499,25 +483,25 @@ def test_insert(self):
         null_index = Index([])
         tm.assert_index_equal(Index(['a']), null_index.insert(0, 'a'))
 
+    @pytest.mark.parametrize("na_val", [np.nan, pd.NaT, None])
```

Comment:
e.g. you can use the nulls_fixture from above

Reply:
I *think* should be including ``None`` because I think we *do* convert it for all but ``object`` dtypes (where it is left alone). so maybe need 2 fixtures. might be tricky to do this in a general way.